libstuff/SLog.cpp

```diff
@@ -50,7 +58,11 @@ string addLogParams(string&& message, const map<string, string>& params) {
             message += " ";
         }
         const auto& param = *next(params.begin(), i);
-        message += param.first + ": '" + param.second + "'";
+        string value = param.second;
+        if (!SContains(PARAMS_WHITELIST, param.first )) {
+            value = "<REDACTED>";
+        }
```

Comment:
Should/can we throw in dev instead? why would anyone pass a param to log that they don't want whitelisted?

Reply:
Love that, yes, let's do it!

Reply:
We don't have `isLive` on a bedrock object here, is it easy to add it @tylerkaraszewski ?